### PR TITLE
Upgrade to sqlalchemy 2.0

### PR DESCRIPTION
**Pull request by amitschang**
This seems faster for pandas to_sql and doesn't have a large impact in code changes. Adding some basic tests for DB load side, though a lot is missed mocking the engine unfortunately, we can still ensure at least we get the the point of executing a query.

### CODE REVIEW

**Review by glemson — COMMENTED** [review]
just a question

File: susdingest/susddatabase.py [hunk 1]
```diff
@@ -70,9 +70,9 @@ def create_schema(self):
             pass
 
     def get_run_id(self, AGENCY, VERSION):
-        sql = f"select id from {self.SCHEMA}.agency_run where agency=%s and version=%s"
+        sql = sqla.text(f"select id from {self.SCHEMA}.agency_run where agency=:a and version=:v")
```
[the remark this answers is not in the document]
Comment (amitschang) [reply susdingest/susddatabase.py:73]: where does the danger come from? I think this just represents a move in the parameter passing form in sqla2 - they expect now a explicit map from key name (`:a`) to value via dict. Though I may be missing what you are referring to...

[the remark this answers is not in the document]
Comment (glemson) [reply susdingest/susddatabase.py:73]: I don't know the sqla.text. if it is equivalent to the usual parameter passing I guess this is all fine indeed. I was thinking about creating a full SQL string by substituting parametsr as strings. but indeed this is different.

**Review by glemson — APPROVED** [review]

File: susdingest/susddatabase.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (amitschang) [reply susdingest/susddatabase.py:73]: ah ok. It seems that sqla went to a slightly changed model for the core execute, and in order to pass text it must be wrapped in the text object, othewise needs to be an executable. So this just updates API

**Pull request merged**